pkg/scheduler/plugins/mig/mig.go

```diff
@@ -39,6 +39,33 @@ type Plugin struct {
 	allocationIndexer cache.Indexer
 }
 
+func (p *Plugin) indexerAdd(obj *instav1alpha1.AllocationClaim) {
+	if p.allocationIndexer == nil {
```

Comment:
the allocationIndexer should be initialized right?

Reply:
hold on, if you are asking if the `allocationIndexer` is initialized or not, it is initialized when the Plugin is initialized. This just refactors the updates to allocationIndexer that already existed.

Reply:
Sure, will do, thanks.